cogboard-app/src/test/kotlin/com/cognifide/cogboard/config/service/VersionServiceTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.cognifide.cogboard.config.service
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class VersionServiceTest {
+
+    @Test
+    fun `Expect -1 when 0-0-0 less than 0-0-1`() {
```

Comment:
if we had `isNewer` method test would be more readable:
`Expect true when newer version given.`